README.md

```diff
@@ -59,6 +65,28 @@ rokus: {
 }
 ```
 
+## Constants
+Each flavor can contain a `constants.yaml`. The strings defined in this file will be added to any `.xml` or `.brs` source files. You can identify the string based on its path in the yaml file. The constants file is idea behind the constants file is similar to Android resource files.
```

Comment:
Might be nice to include a link to https://developer.android.com/guide/topics/resources/accessing-resources.html or something similar for folks who aren't familiar with how Android handles resources

Reply:
I think we should explain how it works instead. Refering users to another system's documentation on how to use your tool is not the best.